### DISCUSSION TITLE
Target date for stable version release

### DISCUSSION THREAD

**Discussion by acheriya** [open-telemetry/opentelemetry-dotnet#4632] [Q&A]
Hi,

Is there any target date for stable version release? 

OpenTelemetry.Instrumentation.SqlClient
OpenTelemetry.Instrumentation.Http

Framework: .Net Framework

Thanks for your contributions.

**Comment by vishweshbankwar** [comment]
`OpenTelemetry.Instrumentation.Http` is expected to be stable by end of this year as Http semantic conventions are expected to be stable soon.
There is no ETA for `OpenTelemetry.Instrumentation.SqlClient`. We will have an ETA once there is an update on db semantic conventions stability.
